unblob/strategies.py

```diff
@@ -38,9 +40,15 @@ def search_chunks_by_priority(path: Path, file: io.BufferedReader) -> List[Chunk
                 real_offset = offset + handler.YARA_MATCH_OFFSET
                 limited_reader = LimitedStartReader(file, real_offset)
                 chunk = handler.calculate_chunk(limited_reader, real_offset)
+
                 # We found some random bytes this handler couldn't parse
                 if chunk is None:
                     continue
+
+                if chunk.end_offset >= file_size:
+                    logger.error("Chunk overflows file", chunk=chunk)
```

Comment:
It would make sense to put the file size to the log context as well.

```suggestion
                    logger.error("Chunk overflows file", chunk=chunk, file_size=format_hex(file_size))
```

`format_hex` since the `Chunk` has its `__repr__` in hex)

Reply:
The `file_size` is already logged when processing the file.